abis/GeneralPurposeFactory.json

```diff
@@ -0,0 +1 @@
9c43b379e4a547e6ad48e4\",\"license\":\"MIT\",\"urls\":[\"bzz-raw://b4a66772187eb01c1ca745ae01f4be401e542fd7f549970bd584bfd79cfe043d\",\"dweb:/ipfs/QmUGUx2yh2TAx499QfySwFeGmpEjcnrQjmfthi9ndKFPQA\"]}},\"version\":1}","metadata":{"compiler":{"version":"0.8.24+commit.e11b9ed9"},"language":"Solidity","output":{"abi":[{"inputs":[],"type":"error","name":"ERC1167FailedCreateClone"},{"inputs":[{"internalType":"address","name":"implementation","type":"address","indexed":false},{"internalType":"address","name":"contractAddress","type":"address","indexed":false}],"type":"event","name":"ContractDeployed","anonymous":false},{"inputs":[{"internalType":"address","name":"implementation","type":"address"},{"internalType":"bytes","name":"initializationData","type":"bytes"},{"internalType":"bytes32","name":"salt","type":"bytes32"}],"stateMutability":"nonpayable","type":"function","name":"createContract","outputs":[{"internalType":"address","name":"","type":"address"}]},{"inputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function","name":"deployedContracts","outputs":[{"internalType":"address","name":"","type":"address"}]}],"devdoc":{"kind":"dev","methods":{"createContract(address,bytes,bytes32)":{"params":{"implementation":"The address of the implementation contract to clone.","initializationData":"The encoded initializer function call.","salt":"A salt for the deterministic deployment."}}},"version":1},"userdoc":{"kind":"user","methods":{"createContract(address,bytes,bytes32)":{"notice":"Deploys a new contract."}},"version":1}},"settings":{"remappings":["@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/","@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/","ds-test/=lib/openzeppelin-contracts-upgradeable/lib/forge-std/lib/ds-test/src/","erc4626-tests/=lib/openzeppelin-contracts-upgradeable/lib/erc4626-tests/","forge-std/=lib/forge-std/src/","openzeppelin-contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/","openzeppelin-contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/"],"optimizer":{"enabled":true,"runs":200},"metadata":{"bytecodeHash":"ipfs"},"compilationTarget":{"src/GeneralPurposeFactory.sol":"GeneralPurposeFactory"},"evmVersion":"paris","libraries":{}},"sources":{"lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/Clones.sol":{"keccak256":"0xd18408af8a91bedb3d56343eeb9b30eb852e6dea93a5e2d5c6db9ca4cb905155","urls":["bzz-raw://061475b7302a732dc598907790f407417551c2b13e89daa57b3698489ef5484b","dweb:/ipfs/QmdX5qmk3VSniSErFuD4aVawQxUi2MuUbP7spPcHddPfcX"],"license":"MIT"},"src/GeneralPurposeFactory.sol":{"keccak256":"0x046299eb611b932c32cdfc6804c3ab00e00338c0469c43b379e4a547e6ad48e4","urls":["bzz-raw://b4a66772187eb01c1ca745ae01f4be401e542fd7f549970bd584bfd79cfe043d","dweb:/ipfs/QmUGUx2yh2TAx499QfySwFeGmpEjcnrQjmfthi9ndKFPQA"],"license":"MIT"}},"version":1},"id":71}
```

Comment:
Can you only add the ABI, and have it formated like this file: https://github.com/snapshot-labs/delegates-subgraph/blob/6718011351028be93fafa46716297c6721215ac2/abis/Token.json

Reply:
0f5687955ecedcfd970161fd9c578daa00e2c091